mrpt_graphslam_2d/CMakeLists.txt

```diff
@@ -18,3 +18,47 @@ if (BUILD_TESTING)
     ${test_output_directory}/${PROJECT_TEST_NAME})
 endif()
 
+set(PROJECT_ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../)
```

Comment:
the PROJECT_ROOT_DIR should have been set in the root CMakeLists directory and then  you can just use it in any of the subdirectories like e.g., here in the mrpt_graphslam_2d subdirectory